municipal-services/ws-services/src/main/java/org/egov/waterconnection/service/WaterServiceImpl.java

```diff
@@ -141,11 +141,14 @@ public List<WaterConnection> createWaterConnection(WaterConnectionRequest waterC
 			throw new CustomException("DUPLICATE_OLD_CONNECTION_NUMBER",
 					"Duplicate Old connection number");
 		}
-		List<WaterConnection> waterConnectionForImisNUmber=getWaterConnectionForImisNUmber(waterConnectionRequest);
-		if(waterConnectionForImisNUmber!=null && waterConnectionForImisNUmber.size()>0)
+		if(waterConnectionRequest.getWaterConnection().getImisNumber()!=null)
```

Comment:
Add null check for old connection no also

Reply:
Added